packages/@okta/vuepress-site/architecture-center/case-studies/companyx/index.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: Landing
+title: User Migration for Company X
+---
+
+# User Migration for Company X
+
+Learn how Company X applied the directory coexistence reference architecture when planning their user migration strategy and the benefits they saw.
+
+## Company Overview
+
+Company X is a Global 2000 company with 200,000 team members worldwide. It uses over 250 SaaS apps, 500 on-premises apps, and 400 cloud-native apps backed by five legacy Identity and Access Management (IAM) systems and multiple Identity Providers (IdPs).
+
+## Related Reference Architecture
+
+<figure class="main-card">
+  <img
+    class="main-card__image"
+    src="/img/architecture/thumbs/usermgmt.png"
+    alt="UserManagement"
+  >
+  <figcaption class="main-card__body">
+    <h2 class="main-card__title">
+      <a
+        class="main-card__main-link"
+        href="/architecture-center/reference-architectures/directory-coexistence"
+      >
+        Directory Coexistence
+      </a>
+    </h2>
+    <div class="main-card__wrapper">
+      <span class="main-card__tag">
+        Architecture paper
+      </span>
+      <span class="main-card__tag">
+        User migration
+      </span>
+      <span class="main-card__tag">
+        Universal Directory
+      </span>
+    </div>
+    <p class="main-card__text">
+      Minimize downtime while migrating user, group, and device profiles to Okta Universal Directory by keeping both source and target directories active during the process. <a class="main-card__link" href="/architecture-center/reference-architectures/directory-coexistence/">Learn more</a>
+    </p>
+  </figcaption>
+</figure>
+
+## The Problem: Too many legacy IAM systems spoil the user administration broth
+
+The company's legacy IAM systems and newer IdPs don't interact well. The number of support tickets related to user onboarding and offboarding is increasing. So are the number of tickets from those users struggling to remember multiple credential combinations as the IAM systems don't synchronize user profiles between them.
+
+### Pain points
+
+* Users need to remember several usernames and passwords to access applications.
+* User profiles aren't synchronized across IAM systems, so they must be manually updated in several places.
+* There is no single sign-on between apps.
+* Administering five legacy IAM systems and trying to coordinate them is difficult and error-prone.
+* Consistent policy-based access management isn't possible as each system implements it differently.
```

Comment:
```suggestion
* Admin users can't set consistent policy-based access management for the company because each IAM system implements access-management policies differently.
```